README.md

```diff
@@ -76,8 +87,8 @@ A list of available versions for `testElasticsearchVersion` can be found on
 
 ### Contributing
 
-If you want to contribute, you find all you need to know in
-[Contributing to Hibernate Search](http://community.jboss.org/wiki/ContributingtoHibernateSearch)
+New contributors are always welcome. We collected some helpful hints on how to get started
+on our website at [Contribute to Hibernate Search](http://hibernate.org/search/contribute/)
 
 ### Source code structure
```

Comment:
There's a lot of stuff not being explained in the README, and whatever we write gets obsolete quickly. I'd rather not add details about stuff that occasional contributors are safe to ignore, the longer readme becomes unnecessarily scary.